checks/cloud/aws/iam/todo.rego

```diff
@@ -0,0 +1,58 @@
+# METADATA
+# title: todo
+# description: todo
+# scope: package
+# schemas:
+# - input: schema["cloud"]
+# custom:
+#   id: AVD-AWS-0346
+#   avd_id: AVD-AWS-0346
+#   provider: aws
+#   service: iam
+#   severity: HIGH
+#   short_code: todo
+#   recommended_action: Create more restrictive S3 policies
+#   input:
+#     selector:
+#     - type: cloud
+#       subtypes:
+#         - service: iam
+#           provider: aws
+package builtin.aws.iam.aws0346
```

Comment:
We can also add a reference to this blog post https://www.aquasec.com/blog/shadow-roles-aws-defaults-lead-to-service-takeover/